### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by princemaple**
`sname`s don't have the `@` and the the host

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,19 +129,19 @@ Let's see how **ExShards** works in distributed fashion.
 Node `a`:
 
 ```
-$ iex --sname a@localhost -S mix
+$ iex --name a@localhost -S mix
```
Comment (cabol) [inline README.md]: If you're going to use `--name` It should be `iex --name a@127.0.0.1 -S mix`

File: README.md [hunk 2]
```diff
@@ -129,19 +129,19 @@ Let's see how **ExShards** works in distributed fashion.
...
 ```
 
 Node `b`:
 
 ```
-$ iex --sname b@localhost -S mix
+$ iex --name b@localhost -S mix
```
Comment (cabol) [inline README.md]: Same comment above

File: README.md [hunk 3]
```diff
@@ -129,19 +129,19 @@ Let's see how **ExShards** works in distributed fashion.
...
 ```
 
 Node `c`:
 
 ```
-$ iex --sname c@localhost -S mix
+$ iex --name c@localhost -S mix
```
Comment (cabol) [inline README.md]: Same comment above

Comment (cabol) [inline README.md]: Besides, in the next step (step 2) you should also change this:

`ExShards.new :mytab, scope: :g, nodes: [:b@localhost, :c@localhost]`

By

`ExShards.new :mytab, scope: :g, nodes: [:"b@127.0.0.1", :"c@127.0.0.1"]`

**Comment by princemaple** [comment]
You are right,. I'll update tomorrow (Melbourne Australia time)

**Comment by cabol** [comment]
@princemaple I think there is nothing wrong to use `a@localhost` with `--sname` ([Distributed Erlang](http://erlang.org/doc/reference_manual/distributed.html)), in this case, if I put the only name (e.g.: `a`), it's completed to `a@your_host_name`. So I added the `@localhost` to avoid that.

Now, I agree it might be better to use `--name` instead of `--sname` in the example, but you need to fix some things, I'll let you come comments!

**Comment by princemaple** [comment]
Sorry @cabol . Just tested, your instructions actually do work. I didn't know that you could have the `@` part in `sname`. Sorry for the noise.

**Comment by cabol** [comment]
@princemaple no problem at all :) !! Actually as I mentioned, it might be better (clearer) to change the example to use `--name`, so think about it, any contribution or suggestion is welcome :) !!

**Comment by princemaple** [comment]
@cabol updated

**Pull request merged**